### PR TITLE
*: bump 0.4.3

### PR DESCRIPTION
**Pull request by BusyJay**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [package]
 name = "tikv-jemallocator"
 # Make sure to update the version in the README as well:
-version = "0.4.1"
+version = "0.4.2"
```
Comment (sticnarf) [inline Cargo.toml:4]: The version inconsistency between `tikv-jemallocator` and `tikv-jemalloc-sys` may confuse people.

If we don't intend to make it consistent, I think should list the versions separately in the CHANGELOG.

Comment (BusyJay) [reply Cargo.toml]: Fixed, now they are consistent.

File: jemalloc-ctl/src/lib.rs [hunk 2]
```diff
@@ -64,7 +64,7 @@
 //!     }
 //! }
 //! ```
-#![deny(missing_docs, broken_intra_doc_links)]
+#![deny(missing_docs, rustdoc::broken_intra_doc_links)]
```
Comment (sticnarf) [inline jemalloc-ctl/src/lib.rs:67]: Just mention this will be rejected as a hard error by an old toolchain, though we never claimed the MSRV.

**Review by sticnarf — APPROVED** [review]
LGTM

File: jemalloc-ctl/src/lib.rs (hunk 2 above)
Comment (BusyJay) [reply jemalloc-ctl/src/lib.rs]: Now they are suppressed instead. It will be updated to latest version when there is a minor version bump.

**Review by sticnarf — APPROVED** [review]

**Pull request merged**